docs/_i18n/en.yml

```diff
@@ -58,3 +64,230 @@ pages:
   xdc_testnet: XDC Testnet
   zksync_testnet: zkSync Testnet
   rotate_dx_certs: Rotate Data Exchange Certificates
+seo:
+  home_title: "Hyperledger FireFly: Decentralized Transaction Processing"
```

Comment:
Is this subtext used elsewhere to describe FireFly? I'm not sure what the "right" string is here yet, but I've never heard FireFly described as a decentralized transaction processor.

Reply:
Naw, we can definitely make it better and more in line with what we've said in the past. I'll get this updated when I get the update content.